### ISSUE TITLE
10 min time out: silently passed

### ISSUE THREAD

**Issue by HenrikBengtsson** [r-lib/revdepcheck#70]
It's not clear that a (10-min) timeout has occurred -  it appears to be silent in the `revdep_check()` output and is reported as an `NA` table entry by `revdep_summary()`, e.g.
```markdown
|package              |version    |error |warning |note |
|:--------------------|:----------|:-----|:-------|:----|
|NA                   |NA         |      |        |     |
|aroma.core           |3.1.0      |      |        |2    |
|NA                   |NA         |      |        |     |
[...]
```

**Comment by gaborcsardi** [comment]
Yeah, timeouts are now marked at the output on the screen, and also by `revdepcheck:::revdep_results` but not in the report. We'll fix this.

**Comment by hadley** [comment]
Duplicate of #41
